notebooks/108-gpu-device/108-gpu-device.ipynb

```diff
@@ -0,0 +1,928 @@
+{
```

Comment:
<p>I'm not sure that regular user has multiple gpu in setup (e.g. my working machine does not have intel gpu at all :)), can we add some checks before this line to run it conditionally or make this code snippet is part of markdown to prevent unexpected errors? I think it is also good to mention about device naming for multiple gpu in query device section</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/openvinotoolkit/openvino_notebooks/pull/788/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='37'/>